config/Dockerfiles/ostree_boot_image/Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM centos:7
+LABEL maintainer "https://github.com/CentOS-PaaS-SIG/ci-pipeline"
+LABEL description="This container is meant to boot a qcow2 image"
+
+ENV HOME=/root
+
+WORKDIR $HOME
+
+COPY epel-release-7-2.noarch.rpm $HOME/
+RUN yum -y install $HOME/epel-release-7-2.noarch.rpm
+
+COPY atomic7-testing.repo /etc/yum.repos.d
+COPY walters-buildtools.repo /etc/yum.repos.d
+
+RUN yum -y install libguestfs libguestfs-tools-c \
+                   git ostree rpm-ostree libvirt-client \
+                   PyYAML python2-setuptools virt-install \
+                   python-pip python-devel gcc net-tools \
+                   openssh-clients sshpass && yum clean all
+
+RUN pip install ara
```

Comment:
Do you want to version lock this like it is here? https://github.com/CentOS-PaaS-SIG/ci-pipeline/blob/master/playbooks/roles/ara/tasks/main.yml

Reply:
Running on the latest centos7.4 it won't work with the version locked at 13.

Reply:
We discussed this and in the future the staging pipeline would catch this so it shouldn't be a concern not to pin it